nautobot/extras/templatetags/plugins.py

```diff
@@ -80,6 +87,17 @@ def plugin_full_width_page(context, obj):
     return _get_registered_content(obj, "full_width_page", context)
 
 
+@register.inclusion_tag("extras/templatetags/plugin_custom_tabs.html", takes_context=True)
+def plugin_custom_tabs(context, obj):
```

Comment:
I'm feeling like since the core template block is named `nav_tabs` that we should instead name this new template tag `plugin_nav_tabs` to maintain the same naming convention.

And instead of naming them `custom_tabs` in the methods/context to also call them `plugin_nav_tabs`.

Reply:
+1 I would like to generally move away from the “custom” branding for anything new. We have too much of it as-is.

Reply:
OK!